client/src/views/Home/JoinParty.jsx

```diff
@@ -76,16 +84,26 @@ function JoinGame({
         >
           Join Game
         </Button>
-      </main>
-      <footer className="app__footer">
         <Button
           onClick={showCreateGameView}
-          type="secondary"
-          className="button-secondary--min-width"
+          className="button-disabled"
+          type="primary"
+          style={{ marginTop: "24px" }}
           icon="+"
         >
           Create Game
         </Button>
+      </main>
+      <footer className="app__footer">
+        <Button onClick={handleHowToPlayModalOpen} className="button-secondary">
+          How To Play
+        </Button>
+        {howToPlayOpen && (
```

Comment:
I think you missed the second part of this comment. You don't need to have any conditional rendering here b/c modal component handles that for you with the isActive prop.